apps/langchain/langgraph.json

```diff
@@ -0,0 +1,8 @@
+{
+  "node_version": "20",
```

Comment:
Wait, didn't we specify 23 before?

Reply:
<img width="515" alt="image" src="https://github.com/user-attachments/assets/a91986fe-8b62-453d-bbdf-33c026a0e576" />

https://langchain-ai.lang.chat/langgraphjs/cloud/reference/cli/#configuration-file

This one is a bit cryptic, prefer to keep it as-is since this was codegen'd as such